SeaPkg/Core/Runtime/SeaResponderReport.c

```diff
@@ -529,6 +579,13 @@ SeaResponderReport (
     goto Exit;
   }
 
+  // CR3 should be consistent across all cores
+  Status = CrossCheckSmBase (CpuIndex, MmBase, (UINTN)(Fixup32Ptr + FIXUP32_CR3_OFFSET) - (UINTN)LocalMmiEntryBase + SMM_HANDLER_OFFSET, sizeof (UINT64));

```

Comment:
Since `Fixup32Ptr` is `UINT32 *`, should this be `sizeof (UINT32)`? Or:

```suggestion
  Status = CrossCheckSmBase (CpuIndex, MmBase, (UINTN)(Fixup32Ptr + FIXUP32_CR3_OFFSET) - (UINTN)LocalMmiEntryBase + SMM_HANDLER_OFFSET, sizeof(Fixup32Ptr[FIXUP32_CR3_OFFSET]));
```